gensrc/proto/persistent_index.proto

```diff
@@ -18,6 +18,7 @@ message IndexWalMetaPB {
 message MutableIndexMetaPB {
     IndexSnapshotMetaPB snapshot = 2;
     repeated IndexWalMetaPB wals = 3;
+    uint32 format_version = 4;
```

Comment:
this feature is not released, so better to arrange format to first, uint32 format_version = 1;

Reply:
> this feature is not released, so better to arrange format to first, uint32 format_version = 1;

done, change in https://github.com/StarRocks/starrocks/pull/5544/commits/a90fdab638c491c937a520442e2ab2c84079f8c9